lib/exercism/team.rb

```diff
@@ -88,6 +88,13 @@ def dismiss(username)
     members.delete(user)
   end
 
+  def dismiss_invitation(username)
+    user = User.find_by_username(username)
```

Comment:
I will fix it, but I don't understand why this is bad ?? Can someone explain this to me please ? 
Thank you !